Hazelcast.Net.sln

```diff
@@ -14,8 +14,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		build-docs.bat = build-docs.bat
 		build-nuget.bat = build-nuget.bat
 		build.bat = build.bat
-		shared.project.props = shared.project.props
```

Comment:
Do we need this change?

Reply:
No, we don't need this change. It was some change done by VS. I will revert it but the question is if it updates solution config it preserves this order or introduces this change (or similar) again...